rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/project/ProjectController.java

```diff
@@ -259,6 +255,16 @@ public ResponseEntity<CollectionModel<EntityModel<Project>>> getProjectsForUser(
             @RequestParam(value = "tag", required = false) String tag,
             @Parameter(description = "Flag to get projects with all details.")
             @RequestParam(value = "allDetails", required = false) boolean allDetails,
+            @Parameter(description = "The version of the project")
+            @RequestParam(value = "version", required = false) String version,
+            @Parameter(description = "The projectResponsible of the project")
+            @RequestParam(value = "projectResponsible", required = false) String projectResponsible,
+            @Parameter(description = "The state of the project")
+            @RequestParam(value = "state", required = false) String ProjectState,
+            @Parameter(description = "The clearingStatus of the project")
+            @RequestParam(value = "clearingStatus", required = false) String ProjectClearingState,
```

Comment:
I meant to change the data type, not the field name.
```suggestion
            @RequestParam(value = "clearingStatus", required = false) ProjectClearingState clearingStatus,
```